bigtable/client/metadata_update_policy.h

```diff
@@ -77,6 +76,25 @@ class MetadataUpdatePolicy {
                        MetadataParamTypes metadata_param_type,
                        std::string table_id);
 
+  /**
+   * Constructor with default metadata pair.
+   *
+   * @param resource_name hierarchical name of resource, including  project id,
+   * instance id and/or table_id.
+   * @param metadata_param_type type to decide prefix for the value of
+   *     x-goog-request-params.
+   * @param cluster_id cluster_id of the snapshot.
+   * @param snapshot_id snapshot_id used in RPC call.
+   */
+  using ClusterId =
```

Comment:
I think you want:

```C++
using ClusterId = noex::TableAdmin::ClusterId;
```

Otherwise they may drift in their definition over time.  Or maybe refactor `ClusterId` and `SnapshotId` to a small header file.